src/prefect/utilities/logging.py

```diff
@@ -121,7 +121,9 @@ def _write_logs(self) -> None:
                 except Exception as exc:
                     # An error occurred on upload, warn and exit the loop (will
                     # retry later)
-                    warnings.warn(f"Failed to write logs with error: {exc!r}")
+                    warnings.warn(f"Failed to write logs with error: {exc!r}, "
```

Comment:
Looks like this needs `black` run on it for formatting; otherwise I'm curious if you want this logged more visibly using a pattern like https://github.com/PrefectHQ/prefect/blob/master/src/prefect/utilities/logging.py#L163-L168 ?

I don't have a preference but just wanted to put that on your radar

Reply:
Ok, styled and added the MAX_BATCH_LOG_LENGTH.